libyul/AST.h

```diff
@@ -35,54 +35,63 @@ namespace solidity::yul
 
 using Type = YulString;
 
-struct TypedName { langutil::SourceLocation location; YulString name; Type type; };
+struct YulAstDebugData {
```

Comment:
Just DebugData? (We are in solidity::yul namespace)

Reply:
Renamed to DebugData